docs push is broken

Caused by `_version.py` stuff introduced in #1373.

```console
Run cd doc && make default
  cd doc && make default
  shell: bash -eo pipefail -l {0}
make -C sphinx default
make[1]: Entering directory '/__w/cclib/cclib/doc/sphinx'
Checking out prod...
git clone https://github.com/cclib/cclib.git _build/cclib_prod
Cloning into '_build/cclib_prod'...
cd _build/cclib_prod; git checkout v1.8.1
Note: switching to 'v1.8.1'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 07260dd0 Merge pull request #1387 from berquist/changelog-1.8.1
Updating prod...
cd _build/cclib_prod; git checkout v1.8.1; git fetch --all --tags
HEAD is now at 07260dd0 Merge pull request #1387 from berquist/changelog-1.8.1
Fetching origin
cd _build/cclib_prod; find . -name "*.pyc" -delete
Generating prod attributes...
ln -fs _build/cclib_prod/cclib cclib
python attributes.py > attributes.rst
Traceback (most recent call last):
  File "/__w/cclib/cclib/doc/sphinx/attributes.py", line 9, in <module>
    import cclib
  File "/__w/cclib/cclib/doc/sphinx/cclib/__init__.py", line 20, in <module>
    from cclib._version import __version__
ModuleNotFoundError: No module named 'cclib._version'
make[1]: *** [Makefile:67: attributes.rst] Error 1
make[1]: Leaving directory '/__w/cclib/cclib/doc/sphinx'
make: *** [Makefile:5: sphinx] Error 2
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,6 +17,7 @@ jobs:
     uses: ./.github/workflows/test_and_package.yml
   docs:
     uses: ./.github/workflows/docs.yml
+    secrets: inherit
   publish:
     needs: [nix, unit-tests, docs]
     uses: ./.github/workflows/publish.yml

